### PR TITLE
[FIX] Table extractor fix for structure tool

### PR DESCRIPTION
**Pull request by Deepak-Kesavan**
## What

- Pass workflow dir path as env to make table extractor work in structure tool

## Why

-

## How

-

## Can this PR break any existing features. If yes, please list possible items. If no, please explain why. (PS: Admins do not merge the PR without this section filled)

-

## Database Migrations

- 

## Env Config

- 

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions

-

## Notes on Testing

-

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

File: worker/src/unstract/worker/clients/docker.py [hunk 1]
```diff
@@ -168,18 +168,20 @@ def get_container_run_config(
...
-                        os.getenv(Env.WORKFLOW_DATA_DIR, ""),
-                        organization_id,
-                        workflow_id,
-                        execution_id,
-                    ),
+                    "source": source_path,
                     "target": os.getenv(Env.TOOL_DATA_DIR, "/data"),
                 }
             )
+            envs[Env.WORKFLOW_DATA_DIR] = source_path
```
Comment (ritwik-g) [inline worker/src/unstract/worker/clients/docker.py:184]: @Deepak-Kesavan why are we adding this here?

**Review by harini-venkataraman — APPROVED** [review]

File: worker/src/unstract/worker/clients/docker.py (hunk 1 above)
Comment (Deepak-Kesavan) [reply worker/src/unstract/worker/clients/docker.py:184]: @ritwik-g It is required for table extractor to work in structure tool.

Comment (ritwik-g) [reply worker/src/unstract/worker/clients/docker.py:184]: @Deepak-Kesavan can you change the ENV name?

Comment (Deepak-Kesavan) [reply worker/src/unstract/worker/clients/docker.py]: Done

**Review by ritwik-g — APPROVED** [review]

**Pull request merged**